tetris.py

```diff
@@ -20,7 +20,7 @@
 PINK = "#FF00FF"
 TEAL = "paleturquoise3"
 
-shape_names = ["I", "L", "L_rev", "O", "E", "Z", "Z_rev"]
+block_letter = ["I", "L", "L_rev", "O", "E", "Z", "Z_rev"]
```

Comment:
The name should end with `s`, as this list contains multiple block letter`s`:

```suggestion
block_letters = ["I", "L", "L_rev", "O", "E", "Z", "Z_rev"]
```

Reply:
I think not because in the get_block_shape we are performing a check 'if self_block_letter == 'I':'

it makes more sense to use a singular here.